<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>csv저장하기</string>
  </property>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>측정 파일 다운로드</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gbx1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>270</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string>저장할 파일명, 대상</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>50</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>기본값 : sensor_data</string>
    </property>
   </widget>
   <widget class="QLabel" name="lab">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>파일 이름을 입력해 주세요</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="led1">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>80</y>
      <width>220</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="led2">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>170</y>
      <width>220</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>140</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>저장할 topic</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>ex) case1,case2</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbx2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>20</y>
     <width>270</width>
     <height>240</height>
    </rect>
   </property>
   <property name="title">
    <string>저장할 센서값</string>
   </property>
   <widget class="QCheckBox" name="cb1">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>60</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>포름 알데히드</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>온습도</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>공기질 센서</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb4">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>180</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>오존</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb5">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>120</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>MOS</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CO2</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbx3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>272</y>
     <width>550</width>
     <height>50</height>
    </rect>
   </property>
   <property name="title">
    <string>저장할  데이터</string>
   </property>
   <widget class="QRadioButton" name="rbn3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>비 측정값</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbn2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>측정값</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbn1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>18</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>전체</string>
    </property>
   </widget>
  </widget>
  <zorder>gbx3</zorder>
  <zorder>gbx2</zorder>
  <zorder>gbx1</zorder>
  <zorder>btn1</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
